app/utils/report/test.py

```diff
@@ -196,9 +177,8 @@ def create_test_report(data, email_format, db_options,
     tests_total = sum(group["total_tests"] for group in groups)
     regr_total = sum(group["regressions"] for group in groups)
 
-    plan_subject = plan_options.get("subject", plan)
-    subject_str = "{}/{} {}: {} tests, {} regressions ({})".format(
-        job, branch, plan_subject, tests_total, regr_total, kernel)
+    subject_str = "{}/{}: {} tests, {} regressions ({})".format(
```

Comment:
That would make a difference in the email report, I guess the subject string should also be passed via the API.

Reply:
I am a bit lost in the code, I will explain what I got:
the hard-coded table is `TEST_PLAN_OPTIONS` , and then there is ` plan_options = TEST_PLAN_OPTIONS.get(plan, {}` and then `plan_subject = plan_options.get("subject", plan)` I thought that they are all related that's why I removed it this way.